app/migrations/versions/463ddd874cb8_.py

```diff
@@ -0,0 +1,34 @@
+"""empty message
+
+Revision ID: 463ddd874cb8
+Revises: fc2eeba8e8e8
+Create Date: 2020-02-17 16:34:47.369834
+
+"""
+from alembic import op
```

Comment:
this migration can't downgrade.. not sure how i feel about that

Reply:
Hard to avoid with type changes like this-- alphanumeric postal codes are pretty common so I think this is worth it.